system/Cache/Handlers/RedisHandler.php

```diff
@@ -112,22 +112,11 @@ public function get(string $key)
             return null;
         }
 
-        switch ($data['__ci_type']) {
-            case 'array':
-            case 'object':
-                return unserialize($data['__ci_value']);
-
-            case 'boolean':
-            case 'integer':
-            case 'double': // Yes, 'double' is returned and NOT 'float'
```

Comment:
This comment is gone. Do we need it?

Reply:
comment is belong to next stmt, this gone as expected, should be manually added if going to re-add.

Reply:
Added the comment manually.